modules/deploy/pages/deployment-option/cloud/managed-connectors/create-snowflake-connector.adoc

```diff
@@ -22,6 +22,8 @@ to hold the data you intend to stream from Redpanda Cloud messages.
 Refer to the https://docs.snowflake.com/en/user-guide/kafka-connector-overview#kafka-connector-limitations[Snowflake Kafka Connector Limitations^]
 documentation for details.
 
+Snowflake tables contain metadata with offset information. If a Redpanda topic is dropped and recreated after the initial sync to Snowflake it can cause issues due to mismatch of offsets, which can result in new data not being sent to Snowflake. When deleting and/or recreating Redpanda topics that have already sent data to Snowflake using the connector, you must delete the corresponding Snowflake table as well.
```

Comment:
```suggestion
Snowflake tables contain metadata with offset information. If a Redpanda topic is dropped and recreated after the initial sync to Snowflake, it can cause issues due to mismatch of offsets. This can result in new data not being sent to Snowflake. If you use this connector to delete or recreate Redpanda topics that have already sent data to Snowflake, you must delete the corresponding Snowflake table as well.
```

Reply:
not sure about this suggestion..   It now reads like the delete/recreate of the topic was done  using the connector which is not the case.

Reply:
the suggestion now reads like the delete/recreate of topic is done via the connector which is not the case.

Reply:
I see what you mean, does this work now?